app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -167,11 +169,28 @@ class HomeFragment : Fragment() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         postponeEnterTransition()
+        changeStrictModePenalty()
```

Comment:
This will only change policy when the browser is started from home screen. However, if we start the browser any other way (app link, i.e: click a link in gmail and open it in Fenix. Another example would be a shortcut on the phone home screen)

Reply:
So I checked different ways to open the app such as opening the lite browser in Gmail, starting Fenix from a Gmail link. the `IntentReceiverActivity` led to the home fragment in both cases. For the shortcut, did you mean just dragging the app to the home screen to create a shortcut and launch the application from there? 
is there another start case I am missing?

Reply:
@sraturi after looking at it for a bit, I think we could go ahead and move the strictMode changes back to `HomeActivity` (right before transitioning to `homefragment`) and see how well it works. in catching violations. How does that sound?